tests/component/test_read_properties.py

```diff
@@ -0,0 +1,163 @@
+
+
+def test__write__get_int32_property__returns_value(read_task: Task):
+    """Test to validate getter for int32 property."""
+    assert read_task.in_stream.offset == 0
+
+
+def test__write__set_int32_property__returns_assigned_value(read_task: Task):
+    """Test to validate setter for int32 property."""
+    read_task.in_stream.offset = 1
+
+    assert read_task.in_stream.offset == 1
+
+
+def test__write__reset_int32_property__returns_default_value(read_task: Task):
+    """Test to validate reset for int32 property."""
+    read_task.in_stream.offset = 2
+
+    del read_task.in_stream.offset
+
+    assert read_task.in_stream.offset == 0
+
+
+def test__write__get_enum_property__returns_value(read_task: Task):
+    """Test to validate getter for enum property."""
+    assert read_task.in_stream.relative_to == ReadRelativeTo.CURRENT_READ_POSITION
+
+
+def test__write__set_enum_property__returns_assigned_value(read_task: Task):
+    """Test to validate setter for enum property."""
+    read_task.in_stream.relative_to = ReadRelativeTo.FIRST_SAMPLE
+
+    assert read_task.in_stream.relative_to == ReadRelativeTo.FIRST_SAMPLE
+
+
+def test__write__reset_enum_property__returns_default_value(read_task: Task):
+    """Test to validate reset for enum property."""
+    read_task.in_stream.relative_to = ReadRelativeTo.FIRST_SAMPLE
+
+    del read_task.in_stream.relative_to
+
+    assert read_task.in_stream.relative_to == ReadRelativeTo.CURRENT_READ_POSITION
+
+
+def test__write__get_float_property__returns_value(read_task: Task):
+    """Test to validate getter for float property."""
+    assert read_task.in_stream.sleep_time == 0.001
+
+
+def test__write__set_float_property__returns_assigned_value(read_task: Task):
+    """Test to validate setter for float property."""
+    read_task.in_stream.sleep_time = 1
+
+    assert read_task.in_stream.sleep_time == 1
+
+
+def test__write__reset_float_property__returns_default_value(read_task: Task):
+    """Test to validate reset for float property."""
+    read_task.in_stream.sleep_time = 3
+
+    del read_task.in_stream.sleep_time
+
+    assert read_task.in_stream.sleep_time == 0.001
+
+
+def test__write__get_uint32_property__returns_value(read_task: Task):
+    """Test to validate getter for uInt32 property."""
+    assert read_task.in_stream.num_chans == 1
+
+
+def test__write__set_uint32_property__returns_assigned_value(read_task: Task):
+    """Test to validate setter for uInt32 property."""
+    read_task.in_stream.logging_file_write_size = 50000
+
+    assert read_task.in_stream.logging_file_write_size == 50000
+
+
+def test__write__reset_uint32_property__returns_default_value(read_task: Task):
+    """Test to validate reset for uInt32 property."""
+    read_task.in_stream.logging_file_write_size = 30000
+
+    del read_task.in_stream.logging_file_write_size
+
+    assert read_task.in_stream.logging_file_write_size == 4294967295
+
+
+def test__write__get_uint64_property__returns_value(read_task: Task):
```

Comment:
Rename test to `....__returns_default_value`. Since we are testing default value